sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/PartitionPublishingProperties.cs

```diff
@@ -36,6 +38,35 @@ public class PartitionPublishingProperties
         ///
         public int? LastPublishedSequenceNumber { get; }
 
+        /// <summary>
+        ///   Determines whether the specified <see cref="System.Object" /> is equal to this instance.
+        /// </summary>
+        ///
+        /// <param name="obj">The <see cref="System.Object" /> to compare with this instance.</param>
+        ///
+        /// <returns><c>true</c> if the specified <see cref="System.Object" /> is equal to this instance; otherwise, <c>false</c>.</returns>
+        ///
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override bool Equals(object obj) => base.Equals(obj);
+
+        /// <summary>
+        ///   Returns a hash code for this instance.
+        /// </summary>
+        ///
+        /// <returns>A hash code for this instance, suitable for use in hashing algorithms and data structures like a hash table.</returns>
+        ///
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override int GetHashCode() => base.GetHashCode();
+
+        /// <summary>
+        ///   Converts the instance to string representation.
+        /// </summary>
+        ///
+        /// <returns>A <see cref="System.String" /> that represents this instance.</returns>
+        ///
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override string ToString() => base.ToString();
```

Comment:
Are these adding any functionality over the base implementation?

Reply:
We've been asked to apply the `[EditorBrowsable]` to these members in public types where they're not particularly useful and not intended to be part of the public API.   The net effect is that this just hides them from auto-completion in compliant editors but doesn't change functionality.